code/controllers/CMSPageHistoryController.php

```diff
@@ -241,120 +308,25 @@ public function VersionsForm() {
 			$hiddenID = new HiddenField('ID', false, "")
 		);
 
-		$actions = new FieldList(
-			new FormAction(
-				'doCompare', _t('CMSPageHistoryController.COMPAREVERSIONS','Compare Versions')
-			),
-			new FormAction(
-				'doShowVersion', _t('CMSPageHistoryController.SHOWVERSION','Show Version')
-			)
-		);
-
-		// Use <button> to allow full jQuery UI styling
-		foreach($actions->dataFields() as $action) $action->setUseButtonTag(true);
-
 		$form = CMSForm::create(
 			$this,
 			'VersionsForm',
 			$fields,
-			$actions
+			new FieldList()
 		)->setHTMLID('Form_VersionsForm');
 		$form->setResponseNegotiator($this->getResponseNegotiator());
 		$form->loadDataFrom($this->getRequest()->requestVars());
 		$hiddenID->setValue($id);
 		$form->unsetValidator();
-		
+
 		$form
 			->addExtraClass('cms-versions-form') // placeholder, necessary for $.metadata() to work
 			->setAttribute('data-link-tmpl-compare', Controller::join_links($this->Link('compare'), '%s', '%s', '%s'))
 			->setAttribute('data-link-tmpl-show', Controller::join_links($this->Link('show'), '%s', '%s'));
-		
-		return $form;
-	}
-	
-	/**
-	 * Process the {@link VersionsForm} compare function between two pages.
-	 *
-	 * @param array
-	 * @param Form
-	 *
-	 * @return html
-	 */
-	public function doCompare($data, $form) {
```

Comment:
I think for the 3.x version of this PR, we can't remove this method (semver). Can you add this back please?